apps/condo/domains/billing/schema/helpers/propertyFinder.js

```diff
@@ -0,0 +1,56 @@
+const { intersection } = require('lodash')
+
+const { loadListByChunks } = require('@condo/domains/common/utils/serverSchema')
+const { Property } = require('@condo/domains/property/utils/serverSchema')
+
+const SYMBOLS_TO_REMOVE_REGEXP = /[!@#$%^&*)(+=_:"'`[\]]/g
+const SPLITTERS_REGEXP = /[,;. ]/
+
+/**
+ * @param {string} addressStr
+ * @returns {string[]}
+ */
+function tokenifyAddress (addressStr) {
+    return addressStr
+        .replace(SYMBOLS_TO_REMOVE_REGEXP, '')
+        .split(SPLITTERS_REGEXP)
+        .filter(Boolean)
+        .filter((x) => x.length > 1)
+}
+
+/**
+ * @param {*} context
+ * @param {string} organizationId
+ * @param {string} address
+ *
+ * @returns {Property}
```

Comment:
Return type is wrong here. Your function returns an array with the following data structure `[Property<Object>, Score<int>]`
It would be great if you indicate that inside `@returns` section

Reply:
It would also be great to see a description of the `score` here

Reply:
You right. The jsdoc was fixed.